open_event/views/admin/models_views/invite.py

```diff
@@ -0,0 +1,23 @@
+from flask_admin import expose
+from flask_admin.contrib.sqla import ModelView
+from flask import request, url_for, redirect
+from ....helpers.data import DataManager
+from ....helpers.data_getter import DataGetter
+from ....helpers import helpers as Helper
+
+
+class InviteView(ModelView):
+
+    @expose('/new/', methods=('GET', 'POST'))
+    def create_view(self, event_id):
+        if request.method == 'POST':
+            email = request.form['email']
+            user = DataGetter.get_user_by_email(email)
+            event = DataGetter.get_event(event_id)
+            if user:
+                DataManager.add_invite_to_event(user.id, event_id)
+                hash = DataGetter.get_invite_by_user_id(user.id).hash
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'hash'](https://www.codacy.com/app/triplez/open-event-orga-server/file/2678420023/issues/source?bid=3233864&fileBranchId=3324174#l19)